website/static/data/events.json

```diff
@@ -1,4 +1,8 @@
 [
+  {
+    "title": "Webinar | From API to Database: A Comprehensive Solution to Perform Online Load Test on Your Distributed System",
```

Comment:
Please make the title shorter. It does not look good in the preview.
![image](https://user-images.githubusercontent.com/36651058/148907905-34385392-3046-4ccb-91c1-b382d329dbde.png)